src/components/Player/Controls/ProgressCtrl.js

```diff
@@ -7,9 +7,23 @@ import { selectors as player } from '../../../redux/modules/player';
 import { getDuration, seek } from '../../../pkg/jwpAdapter/adapter';
 
 export const ProgressCtrl = () => {
+  // https://gist.github.com/gaearon/e7d97cdf38a2907924ea12e4ebdf3c85
+  const [showChild, setShowChild] = useState(false);
+  // Wait until after client-side hydration to show
+  useEffect(() => setShowChild(true), []);
+
+  if (!showChild) {
+    // You can show some kind of placeholder UI here
+    return null;
+  }
+
+  return <Child/>;
```

Comment:
What this do? Why this is good?

Reply:
You're welcome to read [a thorough explanation of Dan Abramov](https://gist.github.com/gaearon/e7d97cdf38a2907924ea12e4ebdf3c85)
You're also welcome to ask "Who this Dan Abramov is?" and I'll answer you

Reply:
Nice peice.

Reply:
Any interesting information about Dan beside what is obvious?

Reply:
No, just wanted to be sure you know the obvious one :)